src/test/scala/dxWDL/util/InstanceTypeDBTest.scala

```diff
@@ -415,4 +415,15 @@ class InstanceTypeDBTest extends FlatSpec with Matchers {
       db.chooseAttrs(None, None, Some(8), Some(false))
     }
   }
+
+  it should "Query returns correct pricing models for org and user" in {
+    val userBilltoProject = DxProject("project-Fq78Pz80jfVXvy7fJV2jzJfB") // project name: public user billto
+    val orgBilltoProject = DxProject("project-Fq78JVj09vZZK3x3J66QXpf6") // project name: public org billto
+
+    val userResult = InstanceTypeDB.query(userBilltoProject, Verbose(on=false, quiet=true, null))
+    val orgResult = InstanceTypeDB.query(orgBilltoProject, Verbose(on=false, quiet=true, null))
+
+    assert(userResult.pricingAvailable)
```

Comment:
Could use please change it to the scalatest likes it?

```
userResult.pricingAvailable shouldBe(true)
```
The same for the next line.